### DISCUSSION TITLE
Peoples don't understand 2FA with TOTP

### DISCUSSION THREAD

**Discussion by glebov21** [community/community#140288] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

By statistic over 34% Americans use password managers. How many peoples can use TOTP? 1%? I think this is bad solution.
Why I can't use password only? I don't want to install another application on my multiple devices.

**Comment by angelogalope** [comment]


**Reply by glebov21** [reply]
> TOTP usage is likely much lower than password managers (34%), possibly around 1%, as it's mainly used by tech dweeb users. Many prefer passwords only for simplicity, but passwords alone are vulnerable to attacks. Other ways like biometrics, email-based 2FA, or hardware keys offer more convenience than TOTP, without needing extra apps. If sticking to just passwords, use strong, unique ones stored in a password manager for better security.

You are chatgpt bot? Github require now only totp, sms(some countries) and hardware keys.

**Comment by tshandler** [comment]
Using only passwords is risky because passwords alone are vulnerable to breaches, phishing, or brute-force attacks. Time-based One-Time Passwords (TOTP) add an extra layer of security, making it much harder for attackers to access accounts, even if they have your password. While TOTP adoption may be lower than password managers, its benefits in securing sensitive data are significant.

You don't necessarily need a separate app for TOTP. Many password managers or services like Google Authenticator and Authy work across devices, consolidating security in one place. Balancing convenience and security is essential, and TOTP provides a critical extra layer.

**Reply by glebov21** [reply]
> Using only passwords is risky because passwords alone are vulnerable to breaches, phishing, or brute-force attacks. Time-based One-Time Passwords (TOTP) add an extra layer of security, making it much harder for attackers to access accounts, even if they have your password. While TOTP adoption may be lower than password managers, its benefits in securing sensitive data are significant.
> 
> You don't necessarily need a separate app for TOTP. Many password managers or services like Google Authenticator and Authy work across devices, consolidating security in one place. Balancing convenience and security is essential, and TOTP provides a critical extra layer.

Excelent time-tested solutions for passwords: 
1) Strong password rules 
2) Wrong password timeout (tries)

Now i was read reviews in google play for Google Authentifcator. **Many peoples lost his keys and access**. I think this is so fragile solution now.
